scripts/generate_mkdocs.py

```diff
@@ -140,6 +143,11 @@ def main() -> None:
 
             f.write(clean_file_content(file_content, title))
 
+    # Format rules docs
+    add_no_escape_text_plugin()
+    for rule_doc in Path("docs/rules").glob("*.md"):
```

Comment:
Probably ought to be cross-platform complaint here
```suggestion
    for rule_doc in (Path("docs") / "rules").glob("*.md"):
```

Reply:
pathlib should resolved this fine, and I was merely copying line 115 above

Reply:
https://github.com/astral-sh/ruff/blob/f7740a8a20e528725daf4fc4f34fa00aec8e4e1e/scripts/generate_mkdocs.py#L111

Reply:
Really it automatically parses names with Unix separators into parts on Windows? 🤯

Reply:
We don't generate documentation on win32 anyway so if it's consistent with the rest of the script sgtm.